### ISSUE TITLE
sudo-prompt requires Rosetta

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#7336]
We use [sudo-prompt](https://www.npmjs.com/package/sudo-prompt) to ask for the users password when needed to execute privileged commands.

On macOS the module uses AppleScript to prompt for the password:

```applescript
do shell script "./sudo-prompt-script" with administrator privileges
```

This script is compiled into an applet, but the applet does not include an `aarch64` binary:

```
./MacOS/applet (for architecture i386):	Mach-O executable i386
./MacOS/applet (for architecture x86_64):	Mach-O 64-bit executable x86_64
```

That means the script will fail unless Rosetta is installed:

```
/bin/sh: ./applet: Bad CPU type in executable
```

The NPM version hasn't been updated in 4 years, so maybe we need to build this module ourselves?

**Comment by jandubois** [comment]
> The NPM version hasn't been updated in 4 years…

Also:

> This repository has been archived by the owner on Sep 23, 2021. It is now read-only.

**Comment by jandubois** [comment]
Fixed by #7415
